### DISCUSSION TITLE
Reference to GeoParquet on developer.ogc.org

### DISCUSSION THREAD

**Discussion by doublebyte1** [opengeospatial/geoparquet#244] [General]
I would like to add a reference to GeoParquet in the "Encodings" section of [developer.ogc.org](developer.ogc.org) .

![geoparquet-dev-website](https://github.com/user-attachments/assets/8f75df69-c0bc-47a5-a8af-152e08e97e9d)

(the links bellow are for https://geoparquet.org/ and https://github.com/opengeospatial/geoparquet/)

Any suggestions regarding the logo and description are welcome!

Thank you

**Comment by kylebarron** [comment]
It's not clear that logo would be permissible: https://github.com/opengeospatial/geoparquet/pull/48

**Reply by jorisvandenbossche** [reply]
I think you could also just use the plain Parquet logo instead, and with the title "GeoParquet" below it, I think that is still clear

**Comment by doublebyte1** [comment]
Thank You, all! That was helpful. The changes are live here (just scroll down to "encodings"): https://developer.ogc.org/#get_started
